src/user/user.ts

```diff
@@ -42,37 +42,42 @@ export class User implements IUser {
    * @param password a secure password to encrypt/decrypt user's data
    * @param options Options to work with user's instance
    */
-  constructor(password: string, options: Partial<UserOptions> = defaultOptions) {
-    this.updatePassword(password, options);
+
+  constructor(
+    password: string,
+    oldPassword?: string,
```

Comment:
constructor should not ask for oldPassword, I don't think we have any case that need this - 100% will be null?